lib/exercism/connection.rb

```diff
@@ -0,0 +1,15 @@
+module Connection
+  def self.included(base)
+    base.extend(Method)
+    base.send(:include, Method)
+  end
+
+  module Method
+    def connect_to(url)
+      Faraday.new(url: url) do |c|
+        c.use Faraday::Response::Logger
+        c.use Faraday::Adapter::NetHttp
+      end
+    end
+  end
```

Comment:
It looks like the goal of this code is to define `connect_to` at both the class and instance levels. I didn't look at this closely in my earlier review, but I think that the desire to define it in both places is a symptom of a possibly incomplete conversion/separation of the `Github` class from being a module-like bag of methods to a class that gets instantiated.

It would be simpler just to reference `self.class.connect_to` in `Github` instance methods. That said, it may be worth revisiting the class responsibility and naming from a high level, and then considering how the existing pieces might be restructured to fit.